From 0f8cb9d4487e6a3e3b5e360e181f009d376d35e1 Mon Sep 17 00:00:00 2001
From: Dmitry Kazakov <dimula73@gmail.com>
Date: Fri, 14 Jun 2024 11:08:38 +0200
Subject: [PATCH] Poison XSIMD's config until the fix is released upstream

We need to detect if this PR is present or not:
https://github.com/xtensor-stack/xsimd/pull/1032
---
 include/xsimd/config/xsimd_config.hpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/xsimd/config/xsimd_config.hpp b/include/xsimd/config/xsimd_config.hpp
index c32a120..4a61b30 100644
--- a/include/xsimd/config/xsimd_config.hpp
+++ b/include/xsimd/config/xsimd_config.hpp
@@ -16,6 +16,8 @@
 #define XSIMD_VERSION_MINOR 0
 #define XSIMD_VERSION_PATCH 0
 
+#define XSIMD_HAS_ARCH_LIST_FIX_PR1032 1
+
 /**
  * high level free functions
  *
-- 
2.23.0.windows.1

